Submitted By: Ryan Oliver <ryan.oliver@pha.com.au>
Date: 2005-04-25
Initial Package Version: 2.1.20
Origin: Ryan Oliver
Description: Fix compilation issues with gcc4

gcc -DHAVE_CONFIG_H -I. -I. -I.. -I../include -I../plugins -I../include -Wall -W -MT client.lo -MD -MP -MF .deps/client.Tpo -c client.c  -fPIC -DPIC -o client.lo
client.c:64: error: static declaration of 'global_callbacks' follows non-static declaration
saslint.h:112: error: previous declaration of 'global_callbacks' was here

--- cyrus-sasl-2.1.20/lib/client.c-orig	2003-11-12 03:26:06.000000000 +1100
+++ cyrus-sasl-2.1.20/lib/client.c	2005-04-25 17:16:56.000000000 +1000
@@ -61,7 +61,7 @@
 
 static cmech_list_t *cmechlist; /* global var which holds the list */
 
-static sasl_global_callbacks_t global_callbacks;
+sasl_global_callbacks_t global_callbacks;
 
 static int _sasl_client_active = 0;
 
